src/dbus/common/types.hpp

```diff
@@ -401,6 +401,60 @@ struct otbrIpCounters
     uint32_t mRxFailure; ///< The number of IPv6 packets failed to receive.
 };
 
+struct otbrChannelQuality
+{
+    uint8_t  mChannel;
+    uint16_t mOccupancy;
+};
+
+struct otbrChildInfo
```

Comment:
Openthread public API has already defined those structures. Is there a good reason why we need this?

Reply:
I've discussed with @bukepo , we don't want to export openthread header to users so we need to define them here,

Reply:
Then, it seems we need to do this for all other openthread public structures in the future.

BTW, what are Cons of reusing openthread API here? @bukepo

Reply:
We want to decouple projects depends on this project, and preventing from them to use headers from this projects' dependencies. Thus we don't need to update all projects when OpenThread's APIs are changed. Take the mac counter as an example, if OpenThread adds a new fields, we don't want to change all projects up to service and database, because the new field may not be used in our products.